apps/linode-marketplace-mc-live-encoder-demo/roles/mcliveencoder/tasks/main.yml

```diff
@@ -38,8 +38,11 @@
     ansible.builtin.command: firewall-cmd --reload
     become: yes
 
+# - name: Start MainConcept Live Encoder
+#   shell: /opt/McLiveEncoder/engine/run.sh &
+#   args:
+#     chdir: /opt/McLiveEncoder/engine/
+
   - name: Start MainConcept Live Encoder
-    shell: /opt/McLiveEncoder/engine/run.sh &
-    args:
-      chdir: /opt/McLiveEncoder/engine/
+    shell: systemctl start --quiet mcliveencodingwebd.service
```

Comment:
Avoid using`shell` to start the service. Please update to use  the `systemd` module to do so.